CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
+## 2.7.4
+
+- fix: response parameters for StoreTempSelectionState
```

Comment:
not 100% sold that this release should be a patch: this is not a bug fix

Reply:
@peol Any thoughts around this?

Reply:
No strong opinion, but releases needs to be done on master with proper credentials I think 😬 The workflow hasn't really been updated for ages.

Reply:
Switched to a minor version instead